app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -425,7 +442,11 @@ class PatientSummaryScreen(
       facility: Facility,
       phoneNumbers: List<TeleconsultPhoneNumber>
   ) {
-    // TODO (SM): Open contact doctor sheet
+    val phoneNumbersArrayList = arrayListOf<TeleconsultPhoneNumber>().apply {
+      addAll(phoneNumbers)
+    }
```

Comment:
Why are we creating a new list here of the exact same thing? We can pass the `phoneNumbers` list directly to the intent factory method, right?

Reply:
No, intent takes in an `ArrayList` and doesn't accept the Kotlin `List`.

Reply:
Then this is a leaky abstraction since the factory method is exposing this detail. We should change the factory method to expect a `List<>` and then convert it to an array list inside that method.

Reply:
Yeah, changed it